README.md

```diff
@@ -31,122 +30,56 @@ npm install semantic-commitlint --save-dev
 
 ## Setup
 
-Add the following to your `package.json`
+Add the following to your `semantic-release` release block (e.g. in the `release` field of your
+`package.json`)
 
 ```json
 {
-  "scripts": {
-    "semantic-commitlint": "semantic-commitlint",
-    "semantic-release": "semantic-release"
-  },
-  "release": {
-    "verifyRelease": ["semantic-commitlint"]
-  }
+  "verifyRelease": ["semantic-commitlint"]
 }
 ```
 
 Setup [semantic-release authentication](https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/ci-configuration.md#ci-configuration) for CI
 
-## Usage
-
-Add the following commands to your CI build process
-
-```bash
-npm run semantic-commitlint -- --ci
-npm run semantic-release
-```
-
-## Local usage
-
-To get early feedback on commit messages you can add the following to a commit hook or your regular set of tests.
-
-```bash
-npm run semantic-commitlint
-```
-
-This allows your project's contributors to get early feedback on their last commit message instead of waiting for CI to fail a build.
+`semantic-commitlint` currently only supports `@commitlint/config-conventional` - please open a PR
+if you need support for other configurations!
 
-* **Last commit only** - this will not validate all new commit messages because Github auth is required to gather commits that have been added since the last release.
+## Usage
 
-## Config
+Simply run `semantic-release` as normal as part of your CI build process
 
-### Skip commits
+## Skip commits
 
-If there are unreleased commits that shouldn't fail a build, then add them to your `package.json` inside the `semanticCommitlint` config.
+If there are unreleased commits that shouldn't fail a build, then add them to the
+`SEMANTIC_COMMITLINT_SKIP` environment variable as comma-separated GIT SHAs. This should be
```

Comment:
Is this something a developer would manually set in the `package.json`, or handled by CI somehow?

Reply:
This is something the developer would set temporarily in the `package.json`, or if they're using Travis, in the environment variables section. It's really only intended to help developers get this set up initially - the expectation is that they implement linting at the appropriate locations to ensure this never happens after the first release with this plugin.